logging/api/QuickStart/QuickStart.cs

```diff
@@ -0,0 +1,76 @@
+﻿/*
+ * Copyright (c) 2016 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+// [START logging_quickstart]
+
+using System;
+// Imports the Google Cloud Logging client library
+using Google.Logging.V2;
+using Google.Logging.Type;
+using System.Collections.Generic;
+using Google.Api;
+
+namespace GoogleCloudSamples
+{
+    public class QuickStart
+    {
+        static void Main(string[] args)
+        {
+            // Your Google Cloud Platform project ID.
+            string projectId = "YOUR-PROJECT-ID";

```

Comment:
Let's add a check detecting YOUR-PROJECT-ID and printing a message that says you need to edit the code.